main.py

```diff
@@ -35,20 +35,18 @@
                 else:
                     print("That letter doesn't appear in the word.")
                     i += 1
-                used_letters.append(letter)
+                    used_letters.append(letter)
             if display == key:
                 print(f"You guessed the word {key}!")
                 print('You survived!')
                 game_results.append('You survived!')
-                used_letters.clear()
                 break
 
         else:
```

Comment:
This
```
while:
else:
```
construct is interesting, I don't think I've seen it in other languages.